packages/oauth/src/index.ts

```diff
@@ -197,14 +200,17 @@ export class InstallProvider {
         const tokensToRefresh: string[] = detectExpiredOrExpiringTokens(authResult, currentUTCSec);
 
         if (tokensToRefresh.length > 0) {
-          const installationUpdates: any = { ...queryResult }; // TODO :: TS
-          const refreshResponses = await this.refreshExpiringTokens(tokensToRefresh);
-
-          // TODO: perhaps this for..of loop could introduce an async delay due to await'ing once for each refreshResp?
-          // Could we rewrite to be more performant and not trigger the eslint warning? Perhaps a concurrent async
-          // map/reduce? But will the return value be the same? Does order of this array matter?
-          // eslint-disable-next-line no-restricted-syntax
-          for (const refreshResp of refreshResponses) {
+          if (queryResult.authVersion !== 'v2') {
```

Comment:
This is not a common situation at all but this can happen if custom `InstallationStore#fetchInstallation()` method returned an unexpected data.